lib/membrane/rtp/parser.ex

```diff
@@ -80,15 +85,27 @@ defmodule Membrane.RTP.Parser do
 
       metadata = Map.put(metadata, :rtp, rtp)
 
-      {{:ok, buffer: {:output, %Buffer{payload: payload, metadata: metadata}}}, state}
+      actions =
+        actions_with_redemand(
+          [buffer: {:output, %Buffer{payload: payload, metadata: metadata}}],
+          state
+        )
+
+      {{:ok, actions}, state}
     else
       :rtcp ->
         case state.rtcp_output_pad do
           nil ->
             {:ok, state}
```

Comment:
redemand should be put here and in general whenever RTCP packet is received, as we don't send a buffer to the `output` pad in that case

Reply:
I think I see what you're saying. I'll refactor so that the `with` comprehension returns a list of actions, and this `handle_process` will always check for redemand.